Add flag that advises using the internal endpoint (#1737)
RedisLabs/redislabs-docs: content/rc/security/vpc-peering.md

Now the VPC Peering request is accepted. Its status in the VPC Peering tab in the Redis Cloud subscription is updated to 'Peer Established'.

> **Make sure to switchover your application connection string to the private endpoint instead of the public endpoint once the peering is established**

Reviewer: ```suggestion
Once peering is established, we recommend switching your application connection string to the private endpoint. 
```
